packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail.tsx

```diff
@@ -82,13 +83,9 @@ const RancherDetail = ({
         <div className="p-3">
           <OsdsTile className="w-full h-full flex-col" inline rounded>
             <div className="flex flex-col w-full">
-              <OsdsText
-                size={ODS_TEXT_SIZE._400}
-                level={ODS_TEXT_LEVEL.heading}
-                color={ODS_THEME_COLOR_INTENT.text}
-              >
+              <CommonTitle typoSize={ODS_TEXT_SIZE._400}>
```

Comment:
why do we have to specify a size for commontitle component ? isn't it supposed to be the same size everywhere ?